SUMMARY.md

```diff
@@ -121,6 +121,7 @@
 - [Application](libs/application/README.md)
   - [Core](libs/application/core/README.md)
   - [Shared Data Providers](libs/application/data-providers/README.md)
+    - [Application Nest Application](libs/application/nest/application/README.md)
```

Comment:
I do agree a bit awkaward. But my thought process was that this lib/module is specific to the "Application" model db stuff and other modules could be introduced later under the application/nest folder.

Reply:
I see. Would it maybe make more sense to then rename the `/nest/`  to `/models/` ?
Then it would be:
```
/libs/application/models/
  |─ application/
  |─ payment/
  |─ someNewModel/
  |─ README.md
```

Reply:
It's interesting to consider how the libs should evolve if/when we are able to further split apart the UI and API of the application system. Maybe it makes sense to have `libs/application-ui` vs `libs/application-api` folders.

I also like our convention of "core" libraries which expose the core functionality of apps. Models especially are awkward in separate NX modules since they can't have relations without circular dependencies.

So `libs/application/api/core`?

Reply:
Hmm so something  like this would be the structure ?
```
/libs/application/api/core/
/libs/payment/api/core/
/libs/someNewModel/api/core
```
And say for instance we want to extract the FileService from the application-api.
```
/libs/application/api/files
```

Reply:
I think more like
```
libs/application/api/core/
  |-- application/
         |-- application.model
         |-- application.service
  |-- someNewModel/
         |-- someNewModel.model
         |-- someNewModel.service
  |-- applicationApiCore.module
```

Yes I think it would make sense to have 
```
libs/application/api/files/
```